package.json

```diff
@@ -116,7 +116,7 @@
     "ngGeolocation": "rjmackay/ngGeolocation",
     "nvd3": "^1.8.4",
     "underscore": "^1.7.0",
-    "ushahidi-platform-pattern-library": "3.7.1-rc.15"
+    "ushahidi-platform-pattern-library": "^3.7.1-rc.20"
```

Comment:
This should be fixed to a specific version, not "^version". The client tightly tied to exact PL versions